modules/apps/frontend-taglib/frontend-taglib-sample-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -7,4 +7,24 @@
 
 <%@ include file="/init.jsp" %>
 
-<liferay-util:include page="/partials/search_paginator.jsp" servletContext="<%= application %>" />
\ No newline at end of file
+<%
+List<TabsItem> tabsItems = frontendSampleDisplayContext.getTabsItems();
+%>
+
+<clay:tabs
+	tabsItems="<%= tabsItems %>"
+>
+
+	<%
+	for (TabsItem tabsItem : tabsItems) {
+	%>
+
+		<clay:tabs-panel>
+			<liferay-util:include page='<%= "/partials/" + tabsItem.get("panelId") + ".jsp" %>' servletContext="<%= application %>" />
+		</clay:tabs-panel>
+
+	<%
+	}
+	%>
+
+</clay:tabs>
```

Comment:
I think `<clay:navigation-bar>` better first here, instead of frontend-only `<clay:tabs>`. We can expect a ton of samples in this module, so scoping samples in their own page should be better for performance and isolation of ongoing errors. 

Optional, not necessary for this PR, would be two level of navigation: first for taglib and second for tag.

Reply:
I was not completely sure about fully understanding the idea of two levels of navigation, but I've added an extra commit changing `<clay:tabs>` by `<clay:navigation-bar>`